flink-core/src/main/java/org/apache/flink/api/common/io/FileInputFormat.java

```diff
@@ -21,10 +21,16 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
+import org.apache.commons.lang3.Validate;
```

Comment:
I'm really sorry that you ran into this, but the community recently decided to use Guava's Preconditions.check() instead of commons lang.
Can you replace that?